iris_markdown_EH.Rmd

```diff
@@ -0,0 +1,123 @@
+---
+title: "Ethan's Iris Analysis in RMarkdown"
+author: "Ethan Hill"
+date: "2020-01-16"
+output: html_document
+---
+
+Hello! Here are the plots and the code for the plots I have generated with the Iris dataset. I have also gone ahead and included the workflow I used to get the final plots...
+
+
+It is always a good habit to set your working directory:
+
+```{r}
+setwd("C:/Users/hille/Documents/Rbootcamp_S2020/Iris-repo/")
+```
+
+
+Next, I've scrambled the iris dataset (gotta make it harder) and stored the scrambled data into the object called mix:
+
+```{r}
+oo <- sample(x=1:150, 150, replace=FALSE)
+mix <- iris[oo,]
+```
+
+
+Once I have the scrambled data stored in the object called "mix" I can store each of the columns as a seperate vector:
```

Comment:
This is really repetitive. You state what youʻre going to do, then after you did it you repeat it and say Next.... Can you please cut repetition?